### ISSUE TITLE
Run an Accessibility Audit

### ISSUE THREAD

**Issue by chznbaum** [757colorcoded/wp-colorcoded#4]
When building a website today, attention should be paid to the accessibility of the site to disabled users, such as those using screen readers or other assistive tech. In Chrome DevTools, [run an accessibility audit](https://developers.google.com/web/tools/lighthouse/) to determine additional tasks or issues needed to ensure our website is accessible.

Please comment with a of items that should be done. These fixes can then be made by yourself or someone else in a Pull Request.
